src/Storage.gs

```diff
@@ -119,4 +119,6 @@ Storage_.prototype.getPrefixedKey_ = function(key) {
   } else {
     return this.prefix_;
   }
-};
\ No newline at end of file
+};
+
+module.exports = {Storage_: Storage_};
```

Comment:
will require your block comment _// Require imports_ for node parsing to comment in during GAS deployment.

Reply:
Makes sense.